assets/model_monitoring/components/src/genai_token_statistics_compute_metrics/run.py

```diff
@@ -27,8 +31,26 @@ def run():
     processor.process(token_df)
     metrics_df = processor.metrics_generator()
     save_spark_df_as_mltable(metrics_df, args.signal_metrics)
-    sample_date = processor.get_latest_run()
-    save_spark_df_as_mltable(sample_date, args.samples_index)
+    sample_data = processor.get_latest_run()
+    save_spark_df_as_mltable(sample_data, args.token_count)
+    samples_index_rows = []
+    samples_index_rows.append({"metric_name": "TokenCount",
+                               "group": "",
+                               "group_dimension": "",
+                               "samples_name": "TokenCount",
+                               "asset": f"azureml_{os.environ['AZUREML_RUN_ID']}_output_data_token_count:1"})
+    spark = init_spark()
+    metadata_schema = StructType(
+        [
+            StructField("metric_name", StringType(), True),
+            StructField("group", StringType(), True),
+            StructField("group_dimension", StringType(), True),
+            StructField("samples_name", StringType(), True),
+            StructField("asset", StringType(), True),
+        ]
+    )
+    sample_df = spark.createDataFrame(samples_index_rows, metadata_schema)
```

Comment:
seems only the asset url is logged in this file, why do we need this? Can't we get the output url from the run metadata?